nck/readers/adobe_reader_2_0.py

```diff
@@ -0,0 +1,311 @@
+# GNU Lesser General Public License v3.0 only
+# Copyright (C) 2020 Artefact
+# licence-information@artefact.com
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+import logging
+import click
+import json
+import datetime
+import requests
+import time
+from itertools import chain
+
+from nck.utils.retry import retry
+from nck.utils.args import extract_args
+from nck.commands.command import processor
+from nck.readers.reader import Reader
+from nck.clients.adobe_client import AdobeClient
+from nck.streams.json_stream import JSONStream
+from nck.helpers.adobe_helper_2_0 import (
+    add_metric_container_to_report_description,
+    get_node_values_from_response,
+    get_item_ids_from_nodes,
+    parse_response,
+)
+
+DATEFORMAT = "%Y-%m-%dT%H:%M:%S"
+API_WINDOW_DURATION = 6
+API_REQUESTS_OVER_WINDOW_LIMIT = 12
+
+logging.basicConfig(level="INFO")
+logger = logging.getLogger()
+
+
+@click.command(name="read_adobe_2_0")
+@click.option("--adobe-client-id", required=True)
+@click.option("--adobe-client-secret", required=True)
+@click.option("--adobe-tech-account-id", required=True)
+@click.option("--adobe-org-id", required=True)
+@click.option("--adobe-private-key-path", required=True)
+@click.option("--adobe-global-company-id", required=True)
+@click.option("--adobe-report-suite-id", required=True)
+@click.option("--adobe-dimension", required=True, multiple=True)
```

Comment:
Tu peux écrire 
```suggestion
@click.option("--adobe-dimension", "dimensions", required=True, multiple=True)
```
pour que ton paramètre s'appelle "dimensions" et non "dimension" dans l'initialisation du reader.

Pas forcément nécessaire mais c'est une remarque